ingest_graph_validator/hydrators/ingest_hydrator.py

```diff
@@ -14,18 +15,16 @@
 
 
 # Example sub_uuid for a small submission (wong retina): 668791ed-deec-4470-b23a-9b80fd133e1c
-
-
 class IngestHydrator(Hydrator):
     """
     DCP Ingest Service Submission hydrator class.
 
     Enables importing of HCA Ingest Service submissions by specifying a Submission ID.
     """
 
-    def __init__(self, graph, submission_uuid):
+    def __init__(self, graph:Graph, submission_uuid):
         super().__init__(graph)
-
+        self.ingest_throttle_period = Config['INGEST_THROTTLE_PERIOD']
```

Comment:
where is this being used?

Reply:
I think it is a remnant to a hardcoded throttling factor that Jacob put in. I extracted it into a constant, and then stopped using it altogether.